app.js

```diff
@@ -1,10 +1,155 @@
+function pop() {
+  const y = document.getElementById('popup');
+
+  if (y.style.display === 'flex') {
+    y.style.display = 'none';
+  } else {
+    y.style.display = 'flex';
+  }
+}
+
+function popDesk() {
+  const z = document.getElementById('desk-popup');
+
+  if (z.style.display === 'flex') {
+    z.style.display = 'none';
+  } else {
+    z.style.display = 'flex';
+  }
+}
+
+const data = {
+  title: 'Project name goes here',
+  badge1: 'HTML/CSS',
+  badge2: 'Ruby on Rails',
+  badge3: 'JavaScript',
+  graphic: './images/Rectangle21.svg',
+  deskgraphic: './images/Group 104.svg',
+  text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+  btntext1: 'See live',
+  btntext2: 'See source',
+};
```

Comment:
According to the project requirements, you're supposed to put the data for all of the projects in an array and generate them dynamically from here. But I can see the data only for one project. Kindly make sure to add an array and the remaining projects as well.

![image](https://user-images.githubusercontent.com/34779928/202270093-d2735a81-46ed-41bc-8c42-54d2f4effc9c.png)

Reply:
- [ ] Kindly create the project section on your HTML dynamically using DOM. The code of the project section must be removed from your index.html and created when the page is loaded. Use the data from your Javascript array to create the whole section. It's one of the project requirements.

<img width="805" alt="Screen Shot 2022-11-16 at 8 46 18 PM" src="https://user-images.githubusercontent.com/47673311/202320842-5499b6c5-626e-4d63-846d-5106489a1aee.png">